Safari FIDO U2F Extension/Info.plist

```diff
@@ -42,7 +42,7 @@
 		</dict>
 	</dict>
 	<key>NSHumanReadableCopyright</key>
-	<string>Copyright © 2016 Yikai Zhao. All rights reserved.</string>
+	<string>Copyright © 2016-2017 Yikai Zhao, et al. All rights reserved. Licensed for use under the MIT license.</string>
```

Comment:
Use `Copyright © 2016-present` to create a copyright notice that's year-agnostic.